Source/Blazorise/Components/NumericEdit/NumericEdit.razor

```diff
@@ -3,11 +3,11 @@
 @inherits Blazorise.BaseTextInput<TValue>
 @if ( IsImmediate && IsDebounce )
 {
-    <input @ref="@ElementRef" id="@ElementId" type="number" name="@NameAttributeValue" inputmode="@InputMode" class="@ClassNames" style="@StyleNames" placeholder="@Placeholder" disabled="@Disabled" readonly="@ReadOnly" size="@VisibleCharacters" step="@StepString" pattern="@Pattern" tabindex="@TabIndex" value="@CurrentValueAsString" @oninput="@OnInputHandler" @onkeydown="@OnKeyDownHandler" @onkeypress="@OnKeyPressHandler" @onkeyup="@OnKeyUpHandler" @onblur="@OnBlurHandler" @onfocus="@OnFocusHandler" @onfocusin="@OnFocusInHandler" @onfocusout="@OnFocusOutHandler" @attributes="@Attributes" />
+    <input @ref="@ElementRef" id="@ElementId" type="number" min="@(minDefined ? Min : null)" max="@(maxDefined ? Max : null)" name="@NameAttributeValue" inputmode="@InputMode" class="@ClassNames" style="@StyleNames" placeholder="@Placeholder" disabled="@Disabled" readonly="@ReadOnly" size="@VisibleCharacters" step="@StepString" pattern="@Pattern" tabindex="@TabIndex" value="@CurrentValueAsString" @oninput="@OnInputHandler" @onkeydown="@OnKeyDownHandler" @onkeypress="@OnKeyPressHandler" @onkeyup="@OnKeyUpHandler" @onblur="@OnBlurHandler" @onfocus="@OnFocusHandler" @onfocusin="@OnFocusInHandler" @onfocusout="@OnFocusOutHandler" @attributes="@Attributes" />
```

Comment:
Better to create new `get` property to get min/max value.

Also, don't forget to update all providers if needed.

Reply:
I tried because it was nullable It seemed like I had to be figuring out how to properly cast it. Probably just making it object would work, would you prefer that?

Reply:
If no other alternative, `object` is fine.

Reply:
Done.